mobile-app/src/components/Card.tsx

```diff
@@ -68,15 +69,21 @@ const styles = StyleSheet.create({
 
 // Component
 
-interface Props {
+export interface Props {
   artwork: {
     title: string;
     artist: string;
     imageURLs: string[];
   };
 }
```

Comment:
I'm not sure that this needs to be exported. Is this interface being used in another file?

Reply:
Thanks for finding those, should be all fixed now.